src/components/home.js

```diff
@@ -1,12 +1,42 @@
 import React, {Component} from 'react';
-import {View} from 'react-native';
+import {View, Text, TouchableOpacity} from 'react-native';
+import Icon from 'react-native-vector-icons/SimpleLineIcons';
+import {Actions} from 'react-native-router-flux';
+import AsyncStorage from '@react-native-community/async-storage';
+
+import Styles from '../style/home';
 
 export default class Home extends Component {
   constructor(props) {
     super(props);
+    this.state = {
+      nickname: '',
+    };
+  }
+
+  componentDidMount() {
+    this.getInfos();
+  }
+
+  async getInfos() {
+    const user = await AsyncStorage.getItem('@user');
+    const {avatar, email, nickname} = JSON.parse(user);
+    this.setState({avatar, email, nickname});
   }
 
   render() {
-    return <View />;
+    const {nickname} = this.state;
+    return (
+      <View style={[Styles.page]}>
+        <View style={[Styles.header]}>
+          <TouchableOpacity onPress={() => Actions.Profile()}>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A function with a name starting with an uppercase letter should only be used as a constructor.](https://app.codacy.com/gh/Full-Moon-Castle/wizard/pullRequest?prid=5422182)